Resources/Audio/_RMC14/Voice/Avali/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["avali_scream"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from Steam mod / Starbound. Made by Steam user: https://steamcommunity.com/id/wonky2 with permission."
+  source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3001955034"
+
+- files: ["avali_1, avali_1_ask, avali_1_exclaim, avali_2, avali_2_ask, avali_2_exclaim"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Taken from Steam mod / Starbound. Made by Steam user: https://steamcommunity.com/id/Nefuki with permission."
+  source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3164757879"
```

Comment:
Are these two NC? It's unclear on whether or not having a Patreon conflicts with the CC NC licenses so we might have to get written permission from the author

Reply:
I see, I'll check